database_scripts/tabels/create_paymentType.sql

```diff
@@ -1,7 +1,8 @@
 use customer_db
 
-create table dbo.PaymentType
+CREATE TABLE [dbo].[PaymentTypes] 
 (
-ID int  NOT NULL PRIMARY KEY,
-[Name] nvarchar(100)
+    [ID] [int] NOT NULL IDENTITY,
+    [Name] [nvarchar](max),
+    CONSTRAINT [PK_dbo.PaymentTypes] PRIMARY KEY ([ID])
 )
```

Comment:
Tu jest ok, tylko musisz dodać NOT NULL przy polach, które są IsRequired(), długości nvarcharów np. [FirstName] [nvarchar](100), zmień `use customer_db` na `use BProject_db`